xml/System/Environment+SpecialFolder.xml

```diff
@@ -99,7 +99,7 @@
       </ReturnValue>
       <MemberValue>48</MemberValue>
       <Docs>
-        <summary>The file system directory that is used to store administrative tools for an individual user. The Microsoft Management Console (MMC) will save customized consoles to this directory, and it will roam with the user. Added in the .NET Framework 4.</summary>
+        <summary>The file system directory that is used to store administrative tools for an individual user. The Microsoft Management Console (MMC) will save customized consoles to this directory, and it will roam with the user. Added in .NET Framework 4.</summary>
```

Comment:
"Added in .NET Framework 4." - do we normally call stuff like this out in the doc? Wouldn't the page simply not exist otherwise?

Reply:
Yes, if you switched the version to .NET Framework 3.5 then these fields wouldn't show up. I'll remove this.